skopt/tests/test_acquisition.py

```diff
@@ -74,3 +95,16 @@ def test_acquisition_gradient():
 
     for acq_func in ["LCB", "PI", "EI"]:
         check_gradient_correctness(X_new, gpr, acq_func, np.max(y))
+
+
+def test_acquisition_per_second():
+    X = [[3], [5], [7]]
+    y = [[1, log(3)], [1, log(5)], [1, log(7)]]
+    mos = MultiOutputSurrogate()
+    mos.fit(X, y)
+
+    X_pred =  [[1], [2], [4], [6], [8], [9]]
+    for acq_func in ["EIps", "PIps"]:
+        vals = _gaussian_acquisition(X_pred, mos, y_opt=1.0, acq_func=acq_func)
+        for fast, slow in zip([0, 1, 2], [5, 4, 3]):
```

Comment:
... to make it a bit more easy to pass the test for the surrogate - while I would expect that most of the time it should learn the right thing, it might not just because data is not too much